### PR TITLE
Upgrade LRP to latest UDK

### PR DESCRIPTION
**Pull request by loneursid**
The WindowsAppRuntime_DLL project is already using the latest UDK which has a fix for brackets surrounding text guids. The LRP also needs to be using that updated version of the UDK.

### CODE REVIEW

File: dev/PushNotifications/PushNotificationsLongRunningTask/packages.config [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.FrameworkUdk.amd64fre" version="10.0.22453.1002-210908-1649.ni-release-svc-reunion2110" targetFramework="native" />
-  <package id="Microsoft.FrameworkUdk.arm64fre" version="10.0.22453.1002-210908-1649.ni-release-svc-reunion2110" targetFramework="native" />
-  <package id="Microsoft.FrameworkUdk.x86fre" version="10.0.22453.1002-210908-1649.ni-release-svc-reunion2110" targetFramework="native" />
+  <package id="Microsoft.FrameworkUdk.amd64fre" version="10.0.22453.1006-210916-1712.ni-release-svc-reunion2110" targetFramework="native" />
```
Comment (sharath2727) [inline dev/PushNotifications/PushNotificationsLongRunningTask/packages.config:3]: why update only the LRP project, please update all the other projects where ever we are using frameworkudk?

Comment (loneursid) [reply dev/PushNotifications/PushNotificationsLongRunningTask/packages.config:3]: Do you know what are these other projects?

Comment (sharath2727) [reply dev/PushNotifications/PushNotificationsLongRunningTask/packages.config:3]: WindowsAppRuntime_DLL?

Comment (loneursid) [reply dev/PushNotifications/PushNotificationsLongRunningTask/packages.config:3]: As stated in the description, this was done when the UDK was ingested.

Comment (sharath2727) [reply dev/PushNotifications/PushNotificationsLongRunningTask/packages.config:3]: Thank you.

**Review by sharath2727 — APPROVED** [review]

**Pull request merged**